tests/install_upgrade_operators/product_install/conftest.py

```
@@ -320,3 +323,23 @@ def cnv_version_to_install_info(is_production_source, ocp_current_version, cnv_i
         pytest.exit(reason="CNV version can't be determined for this run", returncode=INSTALLATION_VERSION_MISMATCH)
     return latest_z_stream
+
+
+@pytest.fixture()
+def default_storage_class(admin_client):
+    # if its not on the matrix - we dont need to test it.
+    default_storage_class_name = py_config["default_storage_class"]
+    if not any(default_storage_class_name in sc_dict for sc_dict in py_config["storage_class_matrix"]):
+        pytest.xfail(f"Storage class {default_storage_class_name} not found in the storage class matrix")
+
+    LOGGER.info(f"Waiting for storage class {default_storage_class_name} to be created")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)